Add Native ScVi Source

Vaedz7 opened the pull request:
Changes
- Added SV to Generator Config
- Added SV programs in Native Programs
- Removed Surprise Trader (Marked as developer only)

Mass Purchase in Separate PR

Mysticial commented:
Sorry for taking so long to get this. I guess to start off, let's remove all the build artifacts. Those don't normally get checked in with the repo - especially if the source is included as well.

Mysticial commented on GeneratorConfig/PokemonSwSh/GodEggDuplication.json:
```diff
@@ -1,25 +0,0 @@
-﻿{
```
Don't delete this file. The program does actually exist, but I think it's hidden.

Vaedz7 replied:
Added back

Mysticial commented on GeneratorConfig/PokemonSV/MassPurchase.json:
```diff
@@ -0,0 +1,32 @@
+    "3-Parameters": [
+        {
+            "00-Type": "BooleanCheckBox",
+            "01-Label": "<b>Pay with LP:</b>,
```
Syntax

Vaedz7 replied:
Added end quote

Vaedz7 commented on GeneratorConfig/PokemonSwSh/DaySkipperUS.json:
```diff
@@ -18,8 +18,8 @@
             "02-Declaration": "const uint16_t REAL_LIFE_YEAR",
             "03-MinValue": 2000,
             "04-MaxValue": 2060,
-            "98-Default": 2022,
```
Changed both US and EU years

Mysticial commented:
You haven't removed all the build artifacts. The entire `obj/` is artifacts. Since you're still learning, whenever a change includes a ton of non-text files that were automatically created, they are usually build files.

Vaedz7 commented:
> You haven't removed all the build artifacts. The entire `obj/` is artifacts. Since you're still learning, whenever a change includes a ton of non-text files that were automatically created, they are usually build files.

Deleted obj folder. Removed build artifacts
